app/Jobs/ImportFacebookSharePosts.php

```diff
@@ -65,10 +66,13 @@ public function tags()
      */
     public function handle(Rogue $rogue)
     {
+        info('STARTING HISTORICAL FB IMPORT');
```

Comment:
@katiecrane I saw you did this in your script and really liked it so stole it ! :)

Reply:
@chloealee happy to share ;)